### ISSUE TITLE
[Bug] ECharts claims to be accessible, but is not keyboard accessible

### ISSUE THREAD

**Issue by kpierro914** [apache/echarts#18585]
### Version

5.4.2

### Link to Minimal Reproduction

https://echarts.apache.org/examples/en/editor.html?c=doc-example%2Faria-decal-simple&edit=1&reset=1

### Steps to Reproduce

1. Visited the demo site and tested multiple chart types with a screen reader in preparation for my company to start using ECharts.
2. Added the following to the options object of the specific chart in the link in the aria section and saw that no role was being added to the main div (same div the label is added to)
label: {description: 'this is a custom label where we can inform users of screen readers to navigate or engage or enable the alternative'}

### Current Behavior

When an aria label is applied to a chart, no role is given to the main div.

### Expected Behavior

Since it is extremely difficult to make SVG's & canvas' keyboard accessible, the short term approach should be when an aria label is applied to any chart, role="img" should also be applied to the main div.

A long term approach may be considered when rendering as an SVG to meet the WCAG 2.1 guidelines (this is different from the ARIA standard). Both are critical to web accessibility.

### Environment

```markdown
- OS:
- Browser:
- Framework:
```

**Comment by helgasoft** [comment]
>  it is extremely difficult to make SVG's & canvas' keyboard accessible

not that difficult actually, see #14706

**Comment by kpierro914** [comment]
The canvas element has support in all browsers and will render on screen but the canvas content will not be accessible to screen readers. With canvas the accessibility has to be added with JavaScript or ARIA on the canvas element itself or using internal fallback content placed within the opening and closing canvas tag. Canvas content is not part of the DOM except for the fallback content.

SVG is a better choice for interactive content and custom controls than Canvas because SVG has internal accessibility semantics and ability to easily add interactivity with JavaScript.

So, how exactly is this easily done? The other thread doesn’t really explain it. It looks like they added key listeners for certain keys and read things aloud to the user when those keys are pressed, but that’s not an all inclusive solution.

**Comment by julianna-langston** [comment]
Hi! I'm one of the developers of chart2music, the library referenced in the other thread.

> It looks like they added key listeners for certain keys and read things aloud to the user when those keys are pressed, but that’s not an all inclusive solution. Some screen reader users are also deaf.

Chart2Music was designed from the ground up to work for all screen reader users. While Chart2Music produces sound, it also leverages the screen reader to "read" out individual values in a chart. (Deaf-blind screen reader users typically use a brailler to physically read what a screen reader is saying, so those users will be able to get those raw data values.)

While the library is optimized for _hearing_ blind users, it still surfaces information to deaf-blind users.
